docs/typescript/populate.md

```diff
@@ -11,7 +11,7 @@ interface Parent {
   name?: string
 }
 const ParentModel = model<Parent>('Parent', new Schema({
-  child: { type: 'ObjectId', ref: 'Child' },
+  child: { type: Types.ObjectId, ref: 'Child' },
```

Comment:
Actually I think it should be Schema.Types.ObjectId

Reply:
That's because we've already imported `Types` from `mongoose`. :)

Reply:
Neh, it is a common misconception. In the interface you use Types.ObjectId but when you instantiate the Schema you use Schema.Types.ObjectId.

Reply:
I've already allowed edits by maintainers. :)

Reply:
```suggestion
  child: { type: Schema.Types.ObjectId, ref: 'Child' },
```